config/config.go

```diff
@@ -1,45 +1,53 @@
 package config
 
-var BindAddr = ":20000"
-
-// The URL of the babbage instance to use.
-var BabbageURL = "http://localhost:8080"
-
-// The URL of the content renderer
-var RendererURL = "http://localhost:20010"
-
-// Dataset routes enabled
-var DatasetRoutesEnabled = false
-
-// The URL of the dataset controller
-var DatasetControllerURL = "http://localhost:20200"
-
-// The URL of the filter dataset controller
-var FilterDatasetControllerURL = "http://localhost:20001"
-
-// The URL of the Geography controller
-var GeographyControllerURL = "http://localhost:23700"
-
-// Geography feature is enabled
-var GeographyEnabled = false
-
-// The URL of Zebedee API
-var ZebedeeURL = "http://localhost:8082"
-
-// The URL of the file downloader service
-var DownloaderURL = "http://localhost:23400"
-
-// The CDN assets path
-var PatternLibraryAssetsPath = "https://cdn.ons.gov.uk/sixteens/e42235b"
-
-// The site domain
-var SiteDomain = "ons.gov.uk"
-
-// Redirect secret
-var RedirectSecret = "secret"
-
-// SQS URL for analytics data
-var SQSAnalyticsURL = ""
-
-// ContentTypeByteLimit respresents the response size at which we stop checking content-type to avoid oom errors
-var ContentTypeByteLimit = 5000000 // 5mb
+import (
+	"github.com/kelseyhightower/envconfig"
+)
+
+// Config represents service coniguration for dp-frontend-router
+type Config struct {
+	BindAddr                   string `envconfig:"BIND_ADDR"`
+	BabbageURL                 string `envconfig:"BABBAGE_URL"`
+	RendererURL                string `envconfig:"RENDERER_URL"`
+	DatasetRoutesEnabled       bool   `envconfig:"DATASET_ROUTES_ENABLED"`
+	DatasetControllerURL       string `envconfig:"DATASET_CONTROLLER_URL"`
+	FilterDatasetControllerURL string `envconfig:"FILTER_DATASET_CONTROLLER_URL"`
+	GeographyControllerURL     string `envconfig:"GEOGRAPHY_CONTROLLER_URL"`
+	GeographyEnabled           bool   `envconfig:"GEOGRAPHY_ENABLED"`
+	ZebedeeURL                 string `envconfig:"ZEBEDEE_URL"`
+	DownloaderURL              string `envconfig:"DOWNLOADER_URL"`
+	PatternLibraryAssetsPath   string `envconfig:"PATTERN_LIBRARY_ASSETS_PATH"`
+	SiteDomain                 string `envconfig:"SITE_DOMAIN"`
+	RedirectSecret             string `envconfig:"REDIRECT_SECRET"`
```

Comment:
For secrets we want to make sure that this value is **never** logged out by the app. You can add this additional tag to the struct which tells the json marshaller to always omit it. Here is an example:

https://github.com/ONSdigital/dp-dataset-api/blob/6b78a084da75982309babf0a433a24fa85064416/config/config.go#L13